sound/hybrid_font.h

```diff
@@ -862,12 +880,18 @@ class HybridFont : public SaberBase, public Looper {
   void StopIdleSound() {
 #ifdef ENABLE_IDLE_SOUND
     RefPtr<BufferedWavPlayer> idlePlayer = GetWavPlayerPlaying(&SFX_idle);
+    RefPtr<BufferedWavPlayer> bgnIdlePlayer = GetWavPlayerPlaying(&SFX_bgnidle);
```

Comment:
We don't need to stop both players, because there is really only one, so we can simplify as:

```
if (!idlePlayer) idlePlayer = GetWavPlayerPlaying(&SFX_bgnidle);
```

There is another problem here, which is that it's possible that bgnidle ends and idle starts right in between these two statements.  It might be enough to just swap them around, like:

```
    // We avoid a race condition by checking for bgnidle before idle.
    RefPtr<BufferedWavPlayer> idlePlayer = GetWavPlayerPlaying(&SFX_bgnidle);
    if (!idlePlayer) idlePlayer = GetWavPlayerPlaying(&SFX_idle);
```